lib/rubocop/cop/lint/parentheses_as_grouped_expression.rb

```diff
@@ -37,6 +37,7 @@ def on_send(node)
         private
 
         def valid_context?(node)
+          return true if first_argument_block_type?(node.first_argument)
```

Comment:
Can you move this line to under the L43 because it should be able to remove the redundant safe navigation operators.

```diff
diff --git a/lib/rubocop/cop/lint/parentheses_as_grouped_expression.rb b/lib/rubocop/cop/lint/parentheses_as_grouped_expression.rb
index 8a7ad837c..ba4dba350 100644
--- a/lib/rubocop/cop/lint/parentheses_as_grouped_expression.rb
+++ b/lib/rubocop/cop/lint/parentheses_as_grouped_expression.rb
@@ -37,17 +37,17 @@ module RuboCop
         private

         def valid_context?(node)
-          return true if first_argument_block_type?(node.first_argument)
           unless node.arguments.one? && first_argument_starts_with_left_parenthesis?(node)
             return true
           end
+          return true if first_argument_block_type?(node.first_argument)

           node.operator_method? || node.setter_method? || chained_calls?(node) ||
             valid_first_argument?(node.first_argument)
         end

         def first_argument_block_type?(first_arg)
-          first_arg&.block_type? || first_arg&.numblock_type?
+          first_arg.block_type? || first_arg.numblock_type?
         end

         def valid_first_argument?(first_arg)
```

Reply:
Indeed it is. Thank you! I updated this PR.